ackhandler/received_packet_handler.go

```diff
@@ -39,31 +39,31 @@ func (h *receivedPacketHandler) ReceivedPacket(packetNumber protocol.PacketNumbe
 		return errInvalidPacketNumber
 	}
 
-	if packetNumber > h.ignorePacketsBelow {
-		if err := h.packetHistory.ReceivedPacket(packetNumber); err != nil {
-			return err
-		}
-	}
-
 	if packetNumber > h.largestObserved {
 		h.largestObserved = packetNumber
 		h.largestObservedReceivedTime = time.Now()
 	}
 
+	if packetNumber <= h.lowerLimit {
+		return nil
```

Comment:
This means we don't call maybeQueueAck anymore, a behavior change from before. I think that should be fine, but we won't count packets towards the 20-packet number.